src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -1182,6 +1181,15 @@ public void handleEndOfLineString(String eol) {
      * A factory to build views for HTML.  The following
      * table describes what this factory will build by
      * default.
+     * <p>
+     * Parsed tags that lack full support are handled differently by the editor.
+     * When the container using HTML tags is editable, all non-supported tags
+     * will be displayed as editable text fields in place of the tags. The contents
+     * within the tags will be handled by the editor as regular text.
```

Comment:
new paragraph - to force a new line here

and "full support" what does that mean ?